mix.exs

```diff
@@ -36,7 +36,7 @@ defmodule Phx.Live.Head.MixProject do
     [
       {:phoenix, ">= 1.6.0"},
       {:phoenix_html, ">= 3.1.0"},
-      {:phoenix_live_view, ">= 0.17.0"},
+      {:phoenix_live_view, ">= 0.19.0"},
```

Comment:
This version will not work in previous versions of LiveView, so I had to bump the minimum version up.